common/tier4_autoware_utils/include/tier4_autoware_utils/geometry/geometry.hpp

```diff
@@ -731,6 +735,70 @@ geometry_msgs::msg::Pose calcInterpolatedPose(
 
   return output_pose;
 }
+
+inline geometry_msgs::msg::Vector3 createVector3(const double x, double y, double z)
+{
+  return geometry_msgs::build<geometry_msgs::msg::Vector3>().x(x).y(y).z(z);
+}
+
+inline geometry_msgs::msg::Twist getTwist(
+  const geometry_msgs::msg::Vector3 & velocity, geometry_msgs::msg::Vector3 & angular)
+{
+  geometry_msgs::msg::Twist twist;
+  twist.linear = velocity;
+  twist.angular = angular;
+  return twist;
+}
+
+inline geometry_msgs::msg::TwistWithCovariance getTwistWithCovariance(
+  const geometry_msgs::msg::Twist & twist)
+{
+  geometry_msgs::msg::TwistWithCovariance twist_with_covariance;
+  twist_with_covariance.twist = twist;
+  return twist_with_covariance;
+}
+
+inline geometry_msgs::msg::TwistWithCovariance getTwistWithCovariance(
+  const geometry_msgs::msg::Vector3 & velocity, geometry_msgs::msg::Vector3 & angular)
```

Comment:
I think it's enough if users write like `getTwistWithCovariance(getTwist(velocity, angular))`.
So, how about removing it?

Reply:
Fixed at [d1c33a3](https://github.com/autowarefoundation/autoware.universe/pull/2049/commits/d1c33a317231386dcdd76d8b05cc29cb9900e757)